adapters-api/src/main/java/io/nosqlbench/engine/api/activityimpl/BaseOpDispenser.java

```diff
@@ -29,59 +29,49 @@
 /**
  * {@inheritDoc}
  * See {@link OpDispenser} for details on how to use this type.
- *
+ * <p>
  * Some details are tracked per op template, which aligns to the life-cycle of the op dispenser.
  * Thus, each op dispenser is where the stats for all related operations are kept.
  *
  * @param <T> The type of operation
  */
 public abstract class BaseOpDispenser<T extends Op, S> implements OpDispenser<T> {
 
-    private final String name;
+    private final String opName;
     protected final DriverAdapter<T, S> adapter;
     private boolean instrument;
     private Histogram resultSizeHistogram;
     private Timer successTimer;
     private Timer errorTimer;
-    private String[] timerStarts = new String[0];
-    private String[] timerStops = new String[0];
+    private final String[] timerStarts;
```

Comment:
Nice catch, maybe we can see what the perf diff on this with conditional is vs 0-length iteration.